sdk/communication/Azure.Communication.Identity/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.1.0-beta.1 (Unreleased)
 
+## 1.1.0-beta.2 (2021-02-22-preview1)
```

Comment:
Let's remove beta.2 and move these under beta.1. Thanks!

Reply:
Done.